docs/prometheus.md

```diff
@@ -0,0 +1,152 @@
+   default](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#prometheusioscrape),
+   Prometheus only discovers ServiceMonitors within its namespace. This should be disabled by setting
+   `serviceMonitorSelectorNilUsesHelmValues` to `false` so that Akri's custom Services and ServiceMonitors can be
+   discovered. Additionally, the Grafana service can be exposed to the host by making it a NodePort service. It may take
+   a minute or so to deploy all the components.
+    ```sh
+    helm install prometheus prometheus-community/kube-prometheus-stack --set grafana.service.type=NodePort --namespace monitoring  --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false
+    ```
+    > **Note**: The Prometheus dashboard can also be exposed to the host by adding `--set
+    > prometheus.service.type=NodePort`.
+
+## Enabling Prometheus in Akri
+The Akri Controller and Agent publish metrics to port 8080 at a `/metrics` endpoint. However, these cannot be accessed
+by Prometheus without first creating Kubernetes Services to expose the Agent and Controller's metrics and secondly
+creating ServiceMonitors, which are custom resources that tell Prometheus which Services to discover. These components
+can all be automatically created and deployed via Helm by setting `--set prometheus.enabled=true` when installing Akri.
+Install Akri and expose the Controller and Agent's metrics to Prometheus by running:
+```sh
+helm repo add akri-helm-charts https://deislabs.github.io/akri/
+helm install akri akri-helm-charts/akri \
+    --set prometheus.enabled=true
+```
+> **Note**: This documentation assumes you are using vanilla Kubernetes. Be sure to reference the [user
+> guide](./user-guide.md) to determine whether the distribution you are using requires crictl path configuration.
+
+## Visualizing metrics with Grafana
+Now that Akri's metrics are being exposed to Prometheus, they can be visualized in Grafana. 
+1. Determine the port that the Grafana Service is running on, specifying the namespace if necessary, and save it for the
+   next step.
+    ```sh
+    kubectl get service/prometheus-grafana  --namespace=monitoring --output=jsonpath='{.spec.ports[?(@.name=="service")].nodePort}' && echo
+    ```
+1. SSH port forwarding can be used to access Grafana. Open a new terminal enter your ssh command to access the machine
+   running Akri and Prometheus followed by the port forwarding request. The following command will use port 50000 on the
+   host. Feel free to change it if it is not available. Be sure to replace `<Grafana Service port>` with the port number
+   outputted in the previous step.
+    ```sh
+    ssh someuser@<IP address> -L 50000:localhost:<Grafana Service port>
+    ```
+1. Navigate to `http://localhost:50000/` and enter Grafana's default username `admin` and password `prom-operator`. Now,
```

Comment:
We might want to put some disclaimer somewhere to let folks know to change the password for security reasons.